### PR TITLE
Add read-excel library version 1.1.6.

### PR DESCRIPTION
**Pull request by igormironchik**
Specify library name and version:  **read-excel/1.1.6**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

I'm the author of this library.

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/read-excel/all/conanfile.py [hunk 1]
```diff
@@ -27,27 +27,6 @@ def _compilers_minimum_version(self):
...
-        content = ""
-        for alias, aliased in targets.items():
-            content += textwrap.dedent("""\
-                if(TARGET {aliased} AND NOT TARGET {alias})
-                    add_library({alias} INTERFACE IMPORTED)
-                    set_property(TARGET {alias} PROPERTY INTERFACE_LINK_LIBRARIES {aliased})
-                endif()
-            """.format(alias=alias, aliased=aliased))
-        tools.save(module_file, content)
-
```
Comment (ericLemanissier) [inline recipes/read-excel/all/conanfile.py:50]: why did you remove all code related to the cmake alias targets ?

File: recipes/read-excel/all/conanfile.py [hunk 2]
```diff
@@ -67,18 +46,11 @@ def source(self):
     def package(self):
         self.copy("COPYING", src=self._source_subfolder, dst="licenses")
         self.copy("*.hpp", src=os.path.join(self._source_subfolder, "read-excel"), dst=os.path.join("include", "read-excel"))
-        self._create_cmake_module_alias_targets(
-            os.path.join(self.package_folder, self._module_file_rel_path),
-            {"read-excel": "read-excel::read-excel"}
-        )
```
Comment (ericLemanissier) [inline recipes/read-excel/all/conanfile.py:73]: same here

File: recipes/read-excel/all/conanfile.py [hunk 3]
```diff
@@ -67,18 +46,11 @@ def source(self):
...
 
     def package_id(self):
         self.info.header_only()
 
     def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "read-excel"
         self.cpp_info.names["cmake_find_package_multi"] = "read-excel"
-        self.cpp_info.builddirs.append(self._module_subfolder)
-        self.cpp_info.build_modules["cmake_find_package"] = [self._module_file_rel_path]
-        self.cpp_info.build_modules["cmake_find_package_multi"] = [self._module_file_rel_path]
```
Comment (ericLemanissier) [inline recipes/read-excel/all/conanfile.py:83]: same here

File: recipes/read-excel/all/conanfile.py (hunk 1 above)
Comment (igormironchik) [reply recipes/read-excel/all/conanfile.py:50]: Because it's not very good to use a library in CMake without namespace. I did so in all my libraries. Minor change for the users, but more standardized.

**Review by madebr — APPROVED** [review]

**Review by ericLemanissier — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]
